### ISSUE TITLE
[12.0] [stock_picking_invoice_link] no related pickings

### ISSUE THREAD

**Issue by Saphar** [OCA/stock-logistics-workflow#522]
After the last updates in Odoo 12.0 branch this plugin doesn't seem to work anymore. I don't see any pickings with the new orders and invoices I create. I see that nothing gets inserted into the "account_invoice_stock_picking_rel" table in the database. Is there a way to find what causes this issue?

**Comment by pedrobaeza** [comment]
There hasn't been any update since the module merge...

**Comment by Saphar** [comment]
Under updates I mean I have the latest code from the branch 12.0. I wonder if it is just some misconfiguration that breaks this.

**Comment by pedrobaeza** [comment]
Please try to reproduce the problem in runbot (admin/admin): https://runbot.odoo-community.org/runbot/154/12.0 and put the detailed steps to reproduce it.

**Comment by Saphar** [comment]
Thank you for the link, it is working in runbot. So it might be something in our configuration, do you have any suggestion where to search for this kind of problem? I am creating sales, deliver the items and create an invoice. It worked fine until yesterday.

**Comment by pedrobaeza** [comment]
Sorry, no clue. Closing the issue then.

**Comment by Saphar** [comment]
I tried again on a separate database, latest Odoo code from git branch 12.0. I installed only the apps Sales, Invoices, stock_picking_invoice_link. I created a sale, delivered the items and generated the invoice, but I cannot see the picking on the invoice. Then I went to the database and inserted these values into the "account_invoice_stock_picking_rel" (1,1) and it appeared on the view the relation to the delivery.

I think somewhere Odoo stopped inserting these values into the database, could that be? On your runbot version it works well, is there any difference?
